modules/installation-guide/partials/proc_installing-che-using-the-che-operator-in-openshift-4-web-console.adoc

```diff
@@ -45,10 +45,10 @@ Operators are a method of packaging, deploying, and managing a Kubernetes applic
 
 . In the *{prod}* pop-up window, click the btn:[Install] button .
 
-. On the *Install Operator* screen, choose following options and validate:
+. On the *Install Operator* screen, specify the following options:
 +
 * *Installation mode*: *A specific {orch-namespace} on the cluster*.
-* *Installed Namespace*: `{prod-short}`.
+* *Installed Namespace*: *Pick an existing namespace* -> `{prod-short}`.
```

Comment:
Just to make sure... shouldn't the value be `{prod-id-short}` here?

Reply:
```suggestion
* *Installed Namespace*: *Pick an existing namespace* -> `{prod-namespace}`.
```

Reply:
@rkratky in downstream it is defined as `:prod-namespace: workspaces` so it is incorrect here, see the screenshot I posed in the PR.

Reply:
We do not pick an existing namespace, we pick the namespace we created befro in the procedure (step 4).

Reply:
Hmm, okay, my mistake.